test_scripts/Smoke/API/021_Speak_SUCCESS.lua

```diff
@@ -0,0 +1,85 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: Speak
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [Speak] SUCCESS on TTS.Speak
+--
+-- Description:
+-- Mobile application sends Speak request with valid parameters to SDL
+--
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+--
+-- Steps:
+-- Application sends Speak request with valid parameters to SDL
+--
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if Speak is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives TTS part of response from HMI with "SUCCESS" result code
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Functions ]]
+local function getRequestParams()
+	return {
+		ttsChunks = {
+			{
+				text ="a",
+				type ="TEXT"
+			}
+		}
+	}
+end
+
+local function speakSuccess(self)
+	local cid = self.mobileSession1:SendRPC("Speak", getRequestParams())
+	EXPECT_HMICALL("TTS.Speak", getRequestParams())
+	:Do(function(_, data)
+			self.hmiConnection:SendNotification("TTS.Started")
+			local function sendSpeakResponse()
+				self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { })
+				self.hmiConnection:SendNotification("TTS.Stopped")
+			end
+			local function sendOnResetTimeout()
+				self.hmiConnection:SendNotification("TTS.OnResetTimeout",
+					{ appID = commonSmoke.getHMIAppId(), methodName = "TTS.Speak" })
+			end
+			RUN_AFTER(sendOnResetTimeout, 9000)
+			RUN_AFTER(sendOnResetTimeout, 18000)
+			RUN_AFTER(sendOnResetTimeout, 24000)
+			RUN_AFTER(sendSpeakResponse, 33000)
```

Comment:
Maybe need to reduce timers, there is no reason to execute test case so long.
Or leave 2 sendOnResetTimeout call to check resetting of timeout to default value and execute test in 20 seconds at least.

Reply:
@dboltovskyi I totaly agree with @GetmanetsIrina in part 

> leave 2 sendOnResetTimeout call to check resetting of timeout to default value

Reply:
@GetmanetsIrina @aderiabin Done in 4f5ab90